src/main/java/org/openelisglobal/notification/dao/NotificationPayloadTemplateDAOImpl.java

```diff
@@ -24,11 +24,13 @@ public NotificationPayloadTemplate getSystemDefaultPayloadTemplateForType(
       NotificationPayloadType type) {
     List<NotificationPayloadTemplate> data;
     try {
-      String sql =
-          "from NotificationPayloadTemplate as npt where npt.type = :type order by npt.id asc";
+      String jpql =
+          "SELECT npt FROM NotificationPayloadTemplate npt WHERE npt.type = :type ORDER BY npt.id"
+              + " ASC";
       Query<NotificationPayloadTemplate> query =
-          entityManager.unwrap(Session.class).createQuery(sql, NotificationPayloadTemplate.class);
-      query.setParameter("type", type.name());
+          entityManager.unwrap(Session.class).createQuery(jpql, NotificationPayloadTemplate.class);
+
+      query.setParameter("type", type);
```

Comment:
This is where you error comes from , Hibernate expects a String and you passed an ENUM . 
change it back to` type.name()`

Reply:
reason why i had removed it was because of:
```
Caused by: java.lang.IllegalArgumentException: Parameter value [TEST_RESULT] did not match expected type [org.openelisglobal.notification.valueholder.NotificationPayloadTemplate$NotificationPayloadType (n/a)]
	at org.hibernate.query.spi.QueryParameterBindingValidator.validate(QueryParameterBindingValidator.java:54)
	at org.hibernate.query.spi.QueryParameterBindingValidator.validate(QueryParameterBindingValidator.java:27)
	at org.hibernate.query.internal.QueryParameterBindingImpl.validate(QueryParameterBindingImpl.java:90)
	at org.hibernate.query.internal.QueryParameterBindingImpl.setBindValue(QueryParameterBindingImpl.java:55)
	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:501)
	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:122)
	at 
```

Reply:
as can be seen in the build error logs on this pr but the app runs well when i build locally